src/Compilers/CSharp/Test/Emit/Emit/EmitErrorTests.cs

```diff
@@ -234,8 +234,8 @@ private static void VerifyEmitDiagnostics(string source2, CSharpCompilation comp
 
                 result.Diagnostics.Verify(expectedDiagnostics.Concat(new[]
                 {
-                    // error CS7038: Failed to emit module 'Test'.
-                    Diagnostic(ErrorCode.ERR_ModuleEmitFailure).WithArguments(compilation2.AssemblyName)
+                    // error CS7038: Failed to emit module 'Test': Unable to determine specific cause of the failure.
+                    Diagnostic(ErrorCode.ERR_ModuleEmitFailure).WithArguments(compilation2.AssemblyName, CodeAnalysisResources.UnableToDetermineSpecificCauseOfFailure)
```

Comment:
>CodeAnalysisResources.UnableToDetermineSpecificCauseOfFailure [](start = 105, length = 61)

This is likely going to cause a problem for Spanish_Unit_Tests once the strings are localized.